fasttld/FastTLDExtract.py

```diff
@@ -49,84 +88,83 @@ def _trie_construct(self, exclude_private_suffix, file_path=''):
         """
         tld_trie = {}
         PublicSuffixList, PrivateSuffixList, AllSuffixList = getPublicSuffixList(file_path)
-        if exclude_private_suffix:
-            SuffixList = PublicSuffixList
-        else:
-            SuffixList = AllSuffixList
+        SuffixList = PublicSuffixList if exclude_private_suffix else AllSuffixList
         for suffix in SuffixList:
             if '.' in suffix:
                 sp = suffix.split('.')
                 sp.reverse()
                 self.nested_dict(tld_trie, sp)
             else:
-                tld_trie.update({
-                    suffix: {
-                        '_END': True
-                    }
-                })
+                tld_trie[suffix] = {'_END': True}
         for key, val in tld_trie.items():
-            if len(tld_trie[key]) == 1:
-                if '_END' in val:
-                    tld_trie.update({
-                        key: True
-                    })
+            if len(val) == 1 and '_END' in val:
+                tld_trie[key] = True
         return tld_trie
-    
+
     def __call__(self, *args, **kwargs):
         return self.extract(*args, **kwargs)
 
-    def extract(self, input, subdomain=True, format=False):
```

Comment:
replaced use of input name as a variable, as input() is a reserved python keyword.